extensions/Dockerfile

```diff
@@ -7,28 +7,7 @@ RUN mkdir /os
 WORKDIR /os
 ADD . .
 ARG OPENSHIFT_CI=0
-RUN --mount=type=secret,id=yumrepos,target=/os/secret.repo <<EOF
-    set -xeuo pipefail
-
-    # fetch repos from in-cluster mirrors if we're running in OpenShift CI
-    if [ "${OPENSHIFT_CI}" != 0 ]; then
-        /run/src/ci/get-ocp-repo.sh ocp.repo
-    fi
-
-    . /etc/os-release
-    # XXX: we can drop the rhcos check once we've dropped the `ocp-rhel-9.6` variant
-    if [ $ID = rhel ] || [ $ID = rhcos ]; then
-        MANIFEST="manifest-rhel-9.6.yaml"
-        EXTENSIONS="extensions-ocp-rhel-9.6.yaml"
-    else
-        MANIFEST="manifest-c9s.yaml"
-        EXTENSIONS="extensions-okd-c9s.yaml"
-    fi
-
-    rpm-ostree compose extensions --rootfs=/ \
-        --output-dir=/usr/share/rpm-ostree/extensions/ \
-        "${MANIFEST}" "${EXTENSIONS}"
-EOF
```

Comment:
There is another `RUN <<EOF` below this line

Reply:
Just reverted that one back to what it was.